js/scrupulous.js

```diff
@@ -372,7 +372,7 @@
           }
 
           if ( $el.is( ':checkbox' ) || $el.is( ':radio' ) ) {
-            elValidity = checkboxValidity( this );
+            elValidity = checkboxValidity(el);
```

Comment:
In order for the `checkboxValidity` function (line 119) to work, it must be passed a reference to an input element. `this` was not such a reference, but `el` is.